functions/notify_slack.py

```diff
@@ -202,6 +222,11 @@ def default_notification(subject, message):
 def filter_message_from_slack(message):
     if message.get('source', "") == "aws.iam" and message.get('detail', {}).get('eventName', '') in ["GenerateCredentialReport", "GenerateServiceLastAccessedDetails", "CreateServiceLinkedRole"]:
       return True
+    elif message.get('source', "") == "aws.ecr":
+      if message.get('detail', {}).get('finding-severity-counts', {}).get('CRITICAL', "") != 0:
```

Comment:
Looks like `finding-severity-counts` will only return values for a severity level if one exists (https://docs.aws.amazon.com/AmazonECR/latest/userguide/ecr-eventbridge.html).

Double checked with an image that only had HIGH vulns:

```
        "imageScanCompletedAt": "2022-06-13T14:25:36-07:00",
        "vulnerabilitySourceUpdatedAt": "2022-06-13T09:34:52-07:00",
        "findingSeverityCounts": {
            "HIGH": 1,
            "MEDIUM": 23,
            "LOW": 5,
            "UNDEFINED": 4
        }
```

Looks like we'd need this to have an evaluation of `!= ""` rather than `!= 0` or just change the default `get(keyname, value)` parameter `value` from `""` to `0` otherwise this will always return False.